xpdacq/analysis.py

```diff
@@ -154,16 +154,18 @@ def save_tiff(headers, dark_subtraction=True, *, max_count=None):
             try:
                 # bluesky only looks for uid it defines
                 dark_search = {'group': 'XPD',
-                               'sc_dark_uid': dark_uid_appended} # the one we need to look up data
+                               'uid': dark_uid_appended} # the one we need to look up for dark
```

Comment:
This file is deprecated. Full functionality is moved to ``xpdAn``. I will check if the same error happen in there.